classes/session/SessionManager.php

```diff
@@ -39,13 +39,16 @@ public function __construct(SessionDAO $sessionDao, PKPRequest $request)
         $this->sessionDao = $sessionDao;
 
         // Configure PHP session parameters
-        ini_set('session.use_trans_sid', 0);
-        ini_set('session.serialize_handler', 'php');
-        ini_set('session.use_cookies', 1);
-        ini_set('session.name', Config::getVar('general', 'session_cookie_name')); // Cookie name
+        ini_set('session.name', Config::getVar('general', 'session_cookie_name'));
         ini_set('session.cookie_lifetime', 0);
         ini_set('session.cookie_path', Config::getVar('general', 'session_cookie_path', $request->getBasePath() . '/'));
         ini_set('session.cookie_domain', $request->getServerHost(null, false));
+        ini_set('session.cookie_httponly', 1);
+        ini_set('session.cookie_samesite', Config::getVar('general', 'same_site', 'lax'));
```

Comment:
Browsers assume `Lax` is the default value when no `SameSite` is defined.

Reply:
I think `lax` should be `Lax`; also in the [OJS PR](https://github.com/pkp/ojs/pull/3696/files#diff-0e9ca26524adfdf100efce10bf37876578f7c505644f2886007be1eccb969e3bR51).

Reply:
AFAICR the value is case-insensitive (I've also checked on Firefox/Chrome), but better to avoid surprises 😂